.kokoro/publish.sh

```diff
@@ -28,4 +28,31 @@ NPM_TOKEN=$(cat $KOKORO_GFILE_DIR/secret_manager/npm_publish_token)
 echo "//wombat-dressing-room.appspot.com/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
 npm install
+
+# Publish the default package
+npm publish --access=public --registry=https://wombat-dressing-room.appspot.com
+
+# Uglify /build to publish a smaller package
+for d in $(find ./build/ -maxdepth 8 -type d)
+do
+  pushd $d
+    for f in *.js; do
+      [ -f "$f" ] || break
+
+      if [ -f "$f.map" ]; then
+        # Keep original .ts source mappings
+        uglifyjs --source-map "content='$f.map', url='$f.map'" "$f" --output "$f"
```

Comment:
I would prefer we pull the logic for minifying into `package.json` scripts, so that when you want the minified version you can just go:

`npm compile-min`

Reply:
This minifies the amount that the publication script deviates from our usual publication script.

Reply:
Good idea. My previous attempt did this - and we can just have a new publish-min script. Will modify